instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -405,28 +467,32 @@ def __call__(self, environ, start_response):
         if response_hook:
             response_hook = functools.partial(response_hook, span, environ)
 
+        start = default_timer()
```

Comment:
Hmm checking the docs, `default_timer()` is `time.perf_counter()`. How does this compare to `time.time_ns()` wall time? I believe we use the latter for span duration right

Reply:
We don't really calculate the duration there. We needed a span start_time and end_time in nanos so `time.time_ns` was a good choice. Here we are interested in the which deals with the relative time. And docs say
> a clock with the highest available resolution to measure a short duration

So this seemed like better choice for measuring duration.

Reply:
Checked with coworkers offline about this. Seems like perf counter/CLOCK_MONOTONIC is the way to go here as it is meant to be wall time with no reference.

Side note, apparently for OTel Java there's a specific clock per-span that measures time-since-span-start using monotonic time